lib/helpers.js

```diff
@@ -92,7 +92,8 @@ _beforeEach.givenModel = function(modelName, attrs, optionalHandler) {
 
   beforeEach(function(done) {
     if(modelName === '__USERMODEL__') {
-      modelName = this.userModel ? this.userModel : 'user';
+      modelName = this.userModel ? this.userModel : 'User';
+      modelKey = modelName;
```

Comment:
@brijs `User` for constructor(Class) name and `user` for instance name, as common.